core/src/main/java/org/kiwix/kiwixmobile/core/page/notes/adapter/NoteListItem.kt

```diff
@@ -1,13 +1,14 @@
 package org.kiwix.kiwixmobile.core.page.notes.adapter
 
 import org.kiwix.kiwixmobile.core.dao.entities.NotesEntity
+import org.kiwix.kiwixmobile.core.dao.entities.NotesRoomEntity
 import org.kiwix.kiwixmobile.core.page.adapter.Page
 import org.kiwix.kiwixmobile.core.reader.ZimFileReader
 
 data class NoteListItem(
   val databaseId: Long = 0L,
   override val zimId: String,
-  override val title: String,
+  override var title: String,
```

Comment:
Why we change this?

Reply:
@MohitMaliDeveloper We are changing this too var because we are setting the title in the tests.